### ISSUE TITLE
Add visitors for ES 2015 node types

### ISSUE THREAD

**Issue by hegemonic** [jsdoc3/jsdoc#624]
When we visit ES 2015 nodes, we currently just [log a warning](https://github.com/jsdoc3/jsdoc/blob/cd639a5c186aa333720bcf9ea189cf9ab4535f26/lib/jsdoc/src/visitor.js#L399) and move on. We need to add real visitors for these node types (or remove the warning for nodes that we don't need to visit):

- [ ] ArrowFunctionExpression
- [ ] ClassBody
- [ ] ClassDeclaration
- [ ] ClassExpression
- [ ] ExportBatchSpecifier
- [ ] ExportDeclaration
- [ ] ExportSpecifier
- [ ] ImportDeclaration
- [ ] ImportSpecifier
- [ ] MethodDefinition
- [ ] ModuleDeclaration
- [x] SpreadElement
- [x] TaggedTemplateExpression
- [x] TemplateElement
- [x] TemplateLiteral

This task should probably be completed in parallel with #623.

**Comment by hegemonic** [comment]
See eslint/espree#10 for information about which ES 2015 features are currently available in the parser.

**Comment by hegemonic** [comment]
On further reflection, I don't think it's useful to track this as its own issue. Closing.
